sdk/identity/identity-common/README.md

```diff
@@ -0,0 +1,39 @@
+# Azure Identity Common client library for JavaScript
```

Comment:
NIT: This isn't a "client" library and we should freely break the README template here. Just "Azure Identity Common Plugin Library" or something like that.

I do think it's nice to have the README, even if we ditch the package.json or other associated files.